.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -8,8 +8,8 @@ on:
       - "docs/sources/**"
   workflow_dispatch:
 jobs:
-  sync:
```

Comment:
I'm not super familiar with GitHub actions; is changing this from 'sync' to 'main' essentially a no-op?

Reply:
Job ids have no particular value unless you're referencing them for things like `needs:` / `needs.` or `jobs.{...}.outputs` or similar.

I didn't mean to change this, I'll undo this change.